protos/fpc/fpc.proto

```diff
@@ -32,9 +31,10 @@ message CCParameters {
 message HostParameters {
     // MSP ID of organization hosting (embracing) the peer with corresponding enclave
     string peer_msp_id = 1;
+    bytes creator = 2;
 
     // the (externally accessible) address of the peer endpoint
-    discovery.Endpoint peer_endpoint = 2;
+    bytes peer_endpoint = 3;
```

Comment:
can we make this a simple string? 

I think that simplifies code inside and outside the enclave a lot!

Reply:
> can we make this a simple string?

Changed.

> I think that simplifies code inside and outside the enclave a lot!

I would say no. In the enclave, strings and binary arrays in protobuf are handled in the same way.
In Go, depending on the use, perhaps you save one conversion.